### PR TITLE
Create alert.css

### PR DESCRIPTION
**Pull request by Radheshyam1234**
Screenshot (45)

### CODE REVIEW

File: Components/alert.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    width:100%;
+    font-size: 1rem;
+    margin:1rem 0;
+    border-radius: 0.3rem;
+    padding:1rem 3rem 1rem 1rem;
+    position: relative;
+}
+.alert-icon{
+    margin-right: 0.7rem;
+    /* height:1.2em;
```
Comment (raghav2110) [inline Components/alert.css]: Better to remove unwanted code rather than commenting.

File: Components/alert.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+.alert-success{
+    background-color: #46d665;
+    color: #023d07;
+    border-left:6px solid rgb(17, 138, 17);
+}
+
+.alert-warning{
+    background-color: #f3ce9b;
```
Comment (raghav2110) [inline Components/alert.css]: Better to use variable for color.

**Review by raghav2110 — COMMENTED** [review]
Overall code looks good.

**Review by ritik-patel05 — COMMENTED** [review]
1. create variables for standard colors of success, danger, warning and use it rather than typing manually. It will be used at other places of websites.
2. Can also create a style guide(variables) for spacing units. And use it, will help you maintain consistency and easy selection rather than typing random values to make it work.
3. Do add a new line at the end of the file. Reference: [link](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)

**Pull request merged**